config_builder/materialize_all.py

```diff
@@ -76,6 +76,15 @@ def main(argv: Sequence[str] | None = None) -> None:
             The root directory of the shared configs.
         """,
     )
+    parser.add_argument(
+        "--ext-package",
+        type=str,
+        action="store",
+        default="",
+        help="""
+            Name of external package to add to jsonnet import paths.
+        """,
+    )
```

Comment:
- Please add a short version of the argument `-p`
- Please support multiple packages to be imported: `action='append'`. It is possible to provide multiple external libraries.